src/utils/hash.ts

```
@@ -360,10 +433,16 @@ class PasswordValidator {
     if (/[a-z]/.test(password)) poolSize += charsets.lowercase;
     if (/[A-Z]/.test(password)) poolSize += charsets.uppercase;
     if (/\d/.test(password)) poolSize += charsets.numbers;
-    if (/[!@#$%^&*()_+\-=```math```{};':"\\|,.<>\/?`~]/.test(password))
-      poolSize += charsets.symbols;
+  if (/[!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?`~]/.test(password)) poolSize += charsets.symbols;
     if (/[^\x00-\x7F]/.test(password)) poolSize += charsets.extended;
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)